__tests__/Routes.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Routes from '../app/js/routes';
+
+describe('Routes component', () => {
+  it('should render without breaking', () => {
+    const wrapper = shallow(<Routes />);
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```

Comment:
I do not think this is an appropriate test suite for the `route.jsx` , and that's because it is not actually rendered, It only returns the component parsed to the `to` prop.
so i'd advise you assert that each Route Component has the appropriate `to` or `path` prop value.

Reply:
Thanks, @Efosaok, that makes sense. Testing the path is a better way to test functionality. Let me update accordingly.

Reply:
Addressed.